client/src/components/App/Content/InvestigationForm/useInvestigationForm.ts

```diff
@@ -190,6 +190,14 @@ const useInvestigationForm = (): useInvestigationFormOutcome => {
     }, [epidemiologyNumber]);
 
     const confirmFinishInvestigation = (epidemiologyNumber: number) => {
+        logger.info({
+            service: Service.CLIENT,
+            severity: Severity.LOW,
+            workflow: 'Ending Investigation',
```

Comment:
Should we put workflow on a constant for easier refactoring down the line?

Reply:
A very good idea Naaman Tsur